package.json

```diff
@@ -21,17 +21,18 @@
     "lint-css": "prettier --write lib/**/*.css",
     "lint-ts": "balena-lint --fix --typescript typings lib tests scripts/clean-shrinkwrap.ts webpack.config.ts",
     "lint": "npm run lint-ts && npm run lint-css",
-    "postinstall": "electron-rebuild -t prod,dev,optional",
+    "preinstall": "bash scripts/ci/preinstall.sh",
```

Comment:
Add `cross-env` or similar instead of `bash`.
On line 17 `flowzone-preinstall-linux` still has `electron-builder.yml` but that file is removed.

Reply:
> Add `cross-env` or similar instead of `bash`.
https://github.com/kentcdodds/cross-env is archived and the package is not gettign any support. Do we really need to use it? My preference would be not to include depracated packages if at all possible, give we already have a ton of those. what does `cross-env` give us over `bash`, if bash is standardised across platforms?

> On line 17 `flowzone-preinstall-linux` still has `electron-builder.yml` but that file is removed.
Cheers, fixed.

Reply:
That is why I wrote "cross-env **_or similar_**"
however from the author:
> So I'm declaring cross-env as finished. No new features will be added (and old features will not likely be removed). I don't plan to do any more than fix security/critical bugs and keep the Node.js support up-to-date.

It still looks like a safe bet.
bash is not available out of the box on many platforms, with a package like this we don't need to write if windows then cmd if linux then bash but only if not zsh, etc and so when a new contributor tries to run `npm i` it will just run even if they don't have bash.